packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -101,6 +101,18 @@ const propTypes = {
    * Function that is called when a collapsible section is selected. Parameters: `onSectionSelect(sectionId)`
    */
   onSectionSelect: PropTypes.func,
+
+  /**
+   * @private
+   * Id of the first row in table
+   */
+  firstRowId: PropTypes.string,
+
+  /**
+   * @private
+   * Id of the last row in table
+   */
+  lastRowId: PropTypes.string,
```

Comment:
This does not match the clinical data grid implementation.  Did we get different requirements?

Reply:
This firstRowId and lastRowId were defined to apply the specific styling for the first and last cells. For a grid without sections we can directly apply through css but with sections we require these values. Similar implementation is done in `clinical-data-grid` https://github.com/cerner/terra-clinical/blob/main/packages/terra-clinical-data-grid/src/DataGrid.jsx#L1058 wherein they use isFirstRow and isLastRow. Here we're using ids instead. And these props dont need to be exposed and will be used only in Cell component.